References
===================

For a complete list of references, you can read `our paper <https://doi.org/10.1101/2023.11.17.567577>`_.

Our Paper 
-------------------------

.. code-block:: text

    @article {Barth2023.11.17.567577,
        author = {Jackson P Barth and Yuqiu Yang and Guanghua Xiao and Xinlei Wang},
        title = {MetaNorm: Incorporating Meta-analytic Priors into Normalization of NanoString nCounter Data},
        elocation-id = {2023.11.17.567577},
        year = {2023},
        doi = {10.1101/2023.11.17.567577},
        publisher = {Cold Spring Harbor Laboratory},
        abstract = {Non-informative or diffuse prior distributions are widely employed in Bayesian data analysis to maintain objectivity. However, when meaningful prior information exists and can be identified, using an informative prior distribution to accurately reflect current knowledge may lead to superior outcomes and great efficiency. We propose MetaNorm, a Bayesian algorithm for normalizing NanoString nCounter gene expression data. MetaNorm is based on RCRnorm, a powerful method designed under an integrated series of hierarchical models that allow various sources of error to be explained by different types of probes in the nCounter system. However, a lack of accurate prior information, weak computational efficiency, and instability of estimates that sometimes occur weakens the approach despite its impressive performance. MetaNorm employs priors carefully constructed from a rigorous meta-analysis to leverage information from large public data. Combined with additional algorithmic enhancements, MetaNorm improves RCRnorm by yielding more stable estimation of normalized values, better convergence diagnostics and superior computational efficiency. R Code for replicating the meta-analysis and the normalization function can be found at github.com/jbarth216/MetaNorm.Competing Interest StatementThe authors have declared no competing interest.},
        URL = {https://www.biorxiv.org/content/early/2023/11/21/2023.11.17.567577},
        eprint = {https://www.biorxiv.org/content/early/2023/11/21/2023.11.17.567577.full.pdf},
        journal = {bioRxiv}
    }


Selected References 
-------------------------

- Jackson Barth, Yuqiu Yang, Guanghua Xiao, and Xinlei Wang. Supplement to "metanorm: Incorporating meta-analytic priors into normalization of nanostring ncounter data". 2022.
- M. J. Bayarri and J. O. Berger. The interplay of bayesian and frequentist analysis. Statistical Science, 19(1):58-80, 2004.
- Arjun Bhattacharya, Alina M. Hamilton, Helena Furberg, Eugene Pietzak, Mark P. Purdue, Melissa A. Troester, Katherine A. Hoadley, and Michael I. Love. An approach for normalization and quality control for nanostring rna expression data. Briefings in Bioinformatics, 22(3), 2021.
- David M. Blei, Alp Kucukelbir, and Jon D. McAuliffe. Variational inference: A review for statisticians. Journal of the American Statistical Association, 112(518):859-877, 2017.
- Xi Chen, Natasha G. Deane, Keeli B. Lewis, Li Jiang, Jing Zhu, M. Kay Washington, and R. Daniel Beauchamp. Comparison of nanostring ncounter® data on ffpe colon cancer samples and affymetrix microarray data on matched frozen tissues. PLoS One, 11(5):e0153784, 2016.
- Dirk Eddelbuettel and Romain Francois. Rcpp: Seamless r and c++ integration. Journal of Statistical Software, 40(8):1-18, 2011.
- Eli Eisenberg and Erez Y. Levanon. Human housekeeping genes, revisited. Trends in Genetics, 29(10):569-574, 2013.
- Gary K. Geiss, Roger E. Bumgarner, Brian Birditt, Timothy Dahl, Naeem Dowidar, Dwayne L. Dunaway, H. P. Fell, Sean Ferree, Renee D. George, Tammy Grogan, Jeffrey J. James, Malini Maysuria, Jeffrey D. Mitton, Paola Oliveri, Jennifer L. Osborn, Tao Peng, Amber L. Ratcliffe, Philippa J. Webster, Eric H. Davidson, Leroy Hood, and Krassen Dimitrov. Direct multiplexed measurement of gene expression with color-coded probe pairs. Nature Biotechnology, 26(3):317-325, 2008.
- Gaoxiang Jia, Xinlei Wang, Qiwei Li, Wei Lu, Ximing Tang, Ignacio Wistuba, and Yang Xie. Rcrnorm: An integrated system of random-coefficient hierarchical regression models for normalizing nanostring ncounter data. The Annals of Applied Statistics, 13(3):1617-47, 2019.
- Gyeong Back Lim, Young-Ae Kim, Jeong-Han Seo, Hee Jin Lee, Gyungyub Gong, and Sung Hee Park. Prediction of prognostic signatures in triple-negative breast cancer based on the differential expression analysis via nanostring ncounter immune panel. BMC Cancer, 20(1), 2020.
- Norikazu Masuda, Tadashi Ohnishi, Shoko Kawamoto, Morito Monden, and Kousaku Okubo. Analysis of chemical modification of rna from formalin-fixed samples and optimization of molecular biology applications for such samples. Nucleic Acids Research, 27(22):4436-4443, 1999.
- NanoString Technologies. nsolvertm 4.0 analysis software.
- Achal Patel, Montserrat Garc ́ıa-Closas, Andrew F. Olshan, Charles M. Perou, Melissa A. Troester, Michael I. Love, and Arjun Bhattacharya. Gene-level germline contributions to clinical risk of recurrence scores in black and white patients with breast cancer. Cancer Research (Chicago, Ill.), 82(1):25-35, 2022.
- Mark A. Perlmutter, Carolyn J. M. Best, John W. Gillespie, Yvonne Gathright, Sergio Gonz ́alez, Alfredo Velasco, W. M. Linehan, Michael Emmert-Buck, and Rodrigo F. Chuaqui. Comparison of snap freezing versus ethanol fixation for gene expression profiling of tissue specimens. The Journal of Molecular Diagnostics, 6(4):371-377, 2004.
- Davide Risso, John Ngai, Terence P Speed, and Sandrine Dudoit. Normalization of rna-seq data using factor analysis of control genes or samples. 32(9):896-902, 2014.
- Daryl Waggott, Kenneth Nature Biotechnology, Chu, Shaoming Yin, Bradly G. Wouters, Fei-Fei Liu, and Paul C. Boutros. Nanostringnorm: an extensible r package for the pre-processing of nanostring mrna and mirna data. Bioinformatics, 28(11):1546-1548, 2012.
- Robert Fred Henry Walter, Robert Werner, Claudia Vollbrecht, Thomas Hager, Elena Flom, Daniel Christian Christoph, Jan Schmeller, Kurt Werner Schmid, Jeremias Wohlschlaeger, and Fabian Dominik Mairinger. Actb, cdkn1b, gapdh, grb2, rhoa and sdcbp were identified as reference genes in neuroendocrine lung cancer via the ncounter technology. PLoS ONE, 11:e0165181, 2016.
- Hong Wang, Craig Horbinski, Hao Wu, Yinxing Liu, Shaoyi Sheng, Jinpeng Liu, Heidi Weiss, Arnold J. Stromberg, and Chi Wang. Nanostringdiff: a novel statistical method for differential expression analysis based on nanostring ncounter data. Nucleic Acids Research, 44(20):e151, 2016.
- Giacomo Zanella and Gareth Roberts. Multilevel linear models, gibbs samplers and multigrid decompositions (with discussion). Bayesian Analysis, 16(4), 2021.